test/unit/serialize.js

```diff
@@ -74,6 +74,9 @@ QUnit.test( "jQuery.param()", function( assert ) {
 	params = { "param1": null };
 	assert.equal( jQuery.param( params, false ), "param1=", "Make sure that null params aren't traversed." );
 
+	params = { "param1": function() {}, "param2": function() { return null; } };
+	assert.equal( jQuery.param( params, false ), "param1=&param2=", "object with method properties return null" );
```

Comment:
`object with method properties return null` -> `Object with method properties that return null`